include/deal.II/hp/refinement.h

```diff
@@ -0,0 +1,355 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2019 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE.md at
+// the top level directory of deal.II.
+//
+// ---------------------------------------------------------------------
+
+#ifndef dealii_hp_refinement_h
+#define dealii_hp_refinement_h
+
+
+#include <deal.II/base/config.h>
+
+DEAL_II_NAMESPACE_OPEN
+
+// forward declarations
+template <typename Number>
+class Vector;
+
+namespace hp
+{
+  template <int dim, int spacedim>
+  class DoFHandler;
+}
+
+namespace hp
+{
+  /**
+   * We supply adaptive methods to align computational ressources with the
+   * complexity of the numerical solution. Error estimates are an appropriate
+   * means of determining where adjustments need to be made.
+   *
+   * However with hp adaptivity, we have two ways to realise these adjustments:
```

Comment:
```suggestion
   * However with hp adaptivity, we have two ways to realize these adjustments:
```
(We try to consistently use American English spelling in deal.II.)